sdk/core/azure-core/test/ut/http_test.cpp

```diff
@@ -121,6 +121,32 @@ namespace Azure { namespace Core { namespace Test {
         (std::pair<std::string, std::string>("valid3", "header3")));
   }
 
+  class ParameterizedTestForHttpVersions
+      : public ::testing::TestWithParam<std::pair<std::int32_t, std::int32_t>> {
+  protected:
+    std::pair<std::int32_t, std::int32_t> httpVersion;
+  };
+
+  INSTANTIATE_TEST_SUITE_P(
+      TestHttp,
+      ParameterizedTestForHttpVersions,
+      ::testing::Values(
+          std::make_pair(0, 9),
+          std::make_pair(1, 0),
+          std::make_pair(1, 1),
```

Comment:
I'd also add 1.5 to the list, but otherwise it looks fine.

Have you verified that this changes code coverage numbers?

Reply:
I checked last week, and the two method calls were covered, indirectly, from one of the curl test that exercised the method calls (as part of keep alive). Therefore, it doesn't change the coverage numbers. Maybe the function call is easily inlined given how small it is.

You can see the yellow/partial cover on line 957, here:
https://github.com/Azure/azure-sdk-for-cpp/issues/5612